client/src/components/advancedSelectWidget/AdvancedSelect.js

```diff
@@ -332,7 +337,36 @@ const AdvancedSelect = ({
                         renderRow={overlayRenderRow}
                         isLoading={isLoading}
                         loaderMessage={
-                          <div style={{ width: "300px" }}>No results found</div>
+                          (createEntityComponent && (
```

Comment:
I would either show result of `createEntityComponent`, or the `OverlayTable` and the `UltimatePagination`, as the latter doesn't make sense at all when creating a new entity.